main.py

```diff
@@ -1,7 +1,32 @@
+import asyncio
+import sys
+import msvcrt
```

Comment:
we do not need the `asyncio` and `sys` libraries anymore.

Reply:
Removed